src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -1575,6 +1575,7 @@ public XMPPError onSessionInfo(JingleSession session, JingleIQ iq)
                     address,
                     bridgeSessionId));
         }
+        listener.participantIceFailed();
```

Comment:
Should we update this even in the "invalid session" case? (I'm not sure what 'invalid session' implies, exactly)

Reply:
I don't know what the "invalid session" case is either. I wanted to keep the semantics closer to "how many times a client reported ICE failed", so this made sense.